src/main/java/com/musala/sdcs/devices/base/IDevice.java

```diff
@@ -0,0 +1,29 @@
+package com.musala.sdcs.devices.base;
+
+public interface IDevice {
```

Comment:
Please add JavaDoc to all methods in the interface. In general, JavaDoc is added to the interface methods, while the classes that implement them only use the `@Override` notation.